package.json

```diff
@@ -44,11 +44,10 @@
     "prebuild": "yarn clean",
     "prepare": "simple-git-hooks && yarn-deduplicate --strategy fewer || exit 0",
     "prerelease": "yarn clean && yarn build",
-    "pretest": "yarn build",
```

Comment:
Remove `pretest `, as local testing takes time. Since it is also checked by CI, there should be no big problem.